sql/moz-fx-data-shared-prod/udf_js/snake_case_columns/udf.sql

```diff
@@ -0,0 +1,69 @@
+CREATE OR REPLACE FUNCTION udf_js.snake_case_columns(input ARRAY<STRING>)
+RETURNS ARRAY<STRING> DETERMINISTIC
+LANGUAGE js
+AS
+  """
```

Comment:
I think you'll want this to be a raw string to avoid issues with having to double escape backslashes in the regular expressions.
```suggestion
  r"""
```